provider/src/main/java/com/cloudera/director/google/GoogleCloudProvider.java

```diff
@@ -38,8 +41,10 @@
 public class GoogleCloudProvider extends AbstractCloudProvider {
 
   public static final String ID = "google";
+  public static boolean featureFlag = Boolean.parseBoolean(System.getenv("DIRECTOR_ENABLE_GOOGLE_CLOUD_SQL_INTEGRATION"));
```

Comment:
I mean 'sudo service cloudera-director-server --full-restart' picks it up from getenv. How can I do it with getProperty?